### ISSUE TITLE
Suppress some warnings

### ISSUE THREAD

**Issue by TheMostDiligent** [microsoft/msvc-code-analysis-action#18]
Is it possible to add an option to suppress some warnings? For example:

```yaml
ignoreRules: C26812
```

**Comment by d-winsor** [comment]
I will encourage users to use standard compiler options `/wd6011` to suppress individual warnings. I will update the readme accordingly.

**Comment by TheMostDiligent** [comment]
This is not a compiler warning though. The one particular I mentioned is this:

```
The enum type 'SOME_ENUM' is unscoped. Prefer 'enum class' over 'enum' (Enum.3).
```

**Comment by d-winsor** [comment]
This compiler option also works for analysis warnings.
